src/features/MentorPage/MentorCard/MentorCardSkills.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+import { ListCardProps } from '../../../../../mentor_card/src/features/MentorPage/ListCard/types';
+import { Text } from '../../../components/CommonTextStyles/Text';
+import { SimpleChip } from '../../../components/Chip';
+
+type Props = {
+  mentorCardData: ListCardProps;
+};
+
+const MentorCardSkills = ({ mentorCardData }: Props) => {
+  if (mentorCardData !== undefined) {
+    return (
+      <Skills>
+        <Text variant="h3" style={{ margin: '3rem 0 0 0' }}>
+          Voin auttaa myös näissä
+        </Text>
+        <SkillChips>
+          {mentorCardData.mentor.skills.map(item => (
+            <SimpleChip key={item.text} text={item.text} />
+          ))}
+        </SkillChips>
+      </Skills>
+    );
+  }
+  return <></>;
+};
```

Comment:
make mentorCardData a required value in the parent  so we can get rid of the check